index.html

```diff
@@ -12,33 +15,64 @@
         background-color: #c7c7c7;
       }
       .produce {
-        height: 18vh;
+		border-bottom: 5px solid;
```

Comment:
Use spaces instead of tabs.  Try to make the indentations consistent - use 6 spaces for the CSS selector, and 8 spaces for anything inside of the braces.